nym-vpn-core/crates/nym-gateway-directory/src/entries/gateway.rs

```diff
@@ -179,12 +204,20 @@ impl TryFrom<nym_validator_client::models::DescribedGateway> for Gateway {
                     .inspect_err(|err| error!("Failed to parse authenticator address: {err}"))
                     .ok()
             });
+        let gateway = nym_topology::gateway::Node::try_from(gateway).ok();
+        let host = gateway.clone().map(|g| g.host);
+        let clients_ws_port = gateway.clone().map(|g| g.clients_ws_port);
```

Comment:
nitpicking: you don't have to be cloning the whole `gateway`, since `u16` is `Copy`